docs/packer.md

```diff
@@ -0,0 +1,32 @@
+# Using Packer and baking AMIs
+
+## Overview
+
+[Packer](http://packer.io/) is well known tool for baking images of any kind.
+We use it to bake our AMIs.
+
+## Prerequisites
+
+* `packer` binary
+* [packer-goss](https://github.com/YaleUniversity/packer-provisioner-goss) plugin
```

Comment:
Not directly related to this PR, but `ansible` should also be listed as a prerequisite

Reply:
Good catch. Added ansible on the list too.